hera_qm/uvflag.py

```diff
@@ -454,8 +490,8 @@ def to_waterfall(self, method='quadmean', keep_pol=True):
             darr = self.metric_array
 
         if self.type == 'antenna':
-            d, w = avg_f(darr, axis=(0, 1), weights=self.weights_array,
-                         returned=True)
+            d, w = qm_utils.collapse(darr, method, axis=(0, 1), weights=self.weights_array,
```

Comment:
I'm generally not a huge fan of single-letter variable names. (It's hard to search for them and sometimes what they represent isn't clear.) It seems that you're pretty consistent about `d` being "data" and `w` being "weights", so I'm inclined to let it be (and since it comes up a lot, it'd be difficult/error prone to replace all instances). I think it's just good coding practice to have in the future.

Reply:
I'll make an issue to address this throughout the repo.